overlay.d/15fcos/usr/lib/systemd/system/coreos-check-cgroups.service

```
@@ -0,0 +1,11 @@
+# This service is used for printing a message if
+# cgroupsv1 is still being used
+[Unit]
+Description=Check if cgroupsv1 is still being used
+ConditionKernelCommandLine=systemd.unified_cgroup_hierarchy=0
```

```suggestion
ConditionControlGroupController=v1
```

TIL about `ConditionControlGroupController`. Seems like a better check to use here. https://www.man7.org/linux/man-pages/man5/systemd.unit.5.html

---

Neat!  I'll give it a try.

---

I notice you did another push.. Did `ConditionControlGroupController=v1` work?